src/cmd/domain.go

```diff
@@ -0,0 +1,233 @@
+package cmd
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"github.com/creasty/defaults"
+	"github.com/opslevel/cli/common"
+	"github.com/opslevel/opslevel-go/v2023"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+	"strings"
+)
+
+var assignSystemCmd = &cobra.Command{
```

Comment:
This command won't work when `opslevel create system` is registered.  We should chat about the design of this one.  I would expect the parent domain to be a parameter on `create system` or `update system` and this command is unncessary

Reply:
Ah ok, yeah I was unsure about this one since I thought it might conflict but thought maybe it would use the right command based on arguments passed in.  But yeah, the create system and update system commands should take a parent input and do the same thing.  Is there a reason we added the AssignSystem mutation then?